cacheops/conf.py

```diff
@@ -78,5 +78,4 @@ def model_profile(model):
     for guess in (app_model, '%s.*' % app, '*.*'):
         profile = model_profiles.get(guess)
         if profile:
-            profile['name'] = app_model
```

Comment:
This function returns the cache profile for a model ie. the cache ttl. The bug was that with wildcard matches the model name would be incorrectly set to first model in the wildcard match which is not necessarily the model being looked up.